### ISSUE TITLE
Add support for `dbt Cloud` as Execution Mode

### ISSUE THREAD

**Issue by jaklan** [astronomer/astronomer-cosmos#888]
Currently `astronomer-cosmos` supports following execution modes:

> **Execution Modes**
>
> Cosmos can run dbt commands using five different approaches, called execution modes:
> 
> 1. **local**: Run dbt commands using a local dbt installation (default)
> 
> 2. **virtualenv**: Run dbt commands from Python virtual environments managed by Cosmos
> 
> 3. **docker**: Run dbt commands from Docker containers managed by Cosmos (requires a pre-existing Docker image)
> 
> 4. **kubernetes**: Run dbt commands from Kubernetes Pods managed by Cosmos (requires a pre-existing Docker image)
> 
> 5. **azure_container_instance**: Run dbt commands from Azure Container Instances managed by Cosmos (requires a pre-existing Docker image)
> 
> The choice of the execution mode can vary based on each user’s needs and concerns. For more details, check each execution mode described below.

It would be great to see `dbt Cloud` in that list as well, so we can automatically [create](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Create%20Job) and [trigger](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Trigger%20Job%20Run) remote jobs using dbt Cloud API under the hood.

**Comment by jaklan** [comment]
We don't have resources for that, I would propose to reach to guys responsible for dbt Cloud Airflow provider: https://github.com/apache/airflow/tree/main/airflow/providers/dbt/cloud

**Comment by tatiana** [comment]
@jaklan I agree this is a relevant feature, but we also have limited bandwidth. If someone from the community is keen on implementing this, we can support them.

**Comment by jaklan** [comment]
Relevant

**Comment by jaklan** [comment]
yes, it's still relevant

**Comment by pankajkoti** [comment]
@jaklan unfortunately, we haven't found anyone who can contribute to this feature.

We're closing this ticket as not planned. But in future, if you find someone who can contribute please reopen the issue and we can assign to them.
